docs/cellery-syntax.md

```diff
@@ -84,6 +84,7 @@ A sample `HttpApiIngress` instance  with inline API definition would be as follo
 cellery:HttpApiIngress helloAPI = {
     port: 9090,
     context: "hello",
+    apiVersion: "1.0.0",
```

Comment:
Please explain how the APIs will be published to the global APIM in this case.

Reply:
Added to the section on exposing ingresses.